tests/parm/model_configure_regional.IN

```diff
@@ -15,7 +15,7 @@ dt_atmos:                225
 cpl:                     .false.
 calendar:                'julian'
 memuse_verbose:          .false.
-atmos_nthreads:          1
+atmos_nthreads:          @[THRD]
 use_hyper_thread:        .false.
 ncores_per_node:         @[TPN]
```

Comment:
Should be removed. As well as in all other model_configure templates.

Reply:
I still see 'memuse_verbose, atmos_nthreads, use_hyper_thread, ncores_per_node' in some model configure files. For example esg_HAFS_v0_hwrf-model_configure.IN.